nav_normal.php

```diff
@@ -0,0 +1,14 @@
+nav_admin.php
```

Comment:
04/08/2021: Make up the login nav for the sign up page that user can access to so that they can make up their account.